tests/vmi_iothreads_test.go

```diff
@@ -133,9 +133,9 @@ var _ = Describe("IOThreads", func() {
 		})
 
 		table.DescribeTable("[ref_id:2065] should honor auto ioThreadPolicy", func(numCpus int, expectedIOThreads int) {
-			if numCpus > availableCPUs {
-				Skip(fmt.Sprintf("Testing environment does not contain a node with required %d CPUs number, the highest detected number is %d", numCpus, availableCPUs))
-			}
+			Expect(numCpus <= availableCPUs).To(BeTrue(),
```

Comment:
nit: `BeNumerically()` is a bit nicer
https://onsi.github.io/gomega/#benumericallycomparator-string-compareto-interface

Reply:
sure, done.